packages/core/scripts/util/file.ts

```diff
@@ -49,6 +53,53 @@ class FileContents implements FileUpdatable {
     this.contents = fs.readFileSync(path, 'utf8');
   }
 
+  public getPaddingFromString(searchString: string): string {
```

Comment:
could both of these be handle by a regex? Something like:
```
const paddingBefore = /\n(\s*)/
const paddingAfter = /(\s*)\n/
const search = new Regexp(paddingBefore.source + searchString + paddingAfter.search, 'm');
```

and then the matching groups for the search would contain both the before and after padding?